docs/extensions/logging-and-monitoring.md

```diff
@@ -156,4 +156,3 @@ If the type of logs exposed in the Plutono instances needs to be changed, it is
 
 * [GitHub Issue Describing the Concept](https://github.com/gardener/gardener/issues/1351)
 * [Exemplary Implementation (Monitoring) for the GCP Provider](https://github.com/gardener/gardener-extension-provider-gcp/blob/master/charts/internal/seed-controlplane/charts/cloud-controller-manager/templates/configmap-observability.yaml)
-* [Exemplary Implementation (Logging) for the OpenStack Provider](https://github.com/gardener/gardener-extension-provider-openstack/blob/master/charts/gardener-extension-provider-openstack/templates/configmap-logging.yaml)
```

Comment:
Can we link https://github.com/gardener/gardener-extension-provider-aws/blob/master/charts/gardener-extension-provider-aws/templates/clusterfilters-logging.yaml and https://github.com/gardener/gardener-extension-provider-aws/blob/master/charts/gardener-extension-provider-aws/templates/clusterparsers-logging.yaml instead?

Reply:
Of course! What would you like these to be called in the references section?

Reply:
Maybe Exemplary Implementation (ClusterFilter), Exemplary Implementation (ClusterParser) ?